README.md

```diff
@@ -328,13 +337,15 @@ const trackErrors = trace(({ input, output, result }) => {
 ```
 
 ## Combining domain functions
-These combinators are useful for composing domain functions. The returning type of all of them is another `DomainFunction` thus allowing further application in more compositions.
+
+These combinators are useful for composing domain functions. They all return another `DomainFunction`, thus allowing further application in more compositions.
```

Comment:
Out of curiosity, is there definitive nomenclature for the input DFs as well as the resulting composed DF? There are parts of the docs that would be more clear (and easier to explain) if I could refer to them as `parent`/`children` or similar

Reply:
We never thought about it because we use to see them all as just DF.
I get your point though, maybe we could call it: composite df